scripts/src/mock_client.ts

```diff
@@ -1543,7 +1567,9 @@ async function main() {
 	await v1P2p();
```

Comment:
uncomment all the tests.
Maybe we can convert this into jest and choose to run a single function from command line